src/test/resources/capabilities-local-parallel-browsers.yml

```diff
@@ -0,0 +1,55 @@
+testEndpoint: http://bs-local.com:3000/
+
+driverType: cloudDriver
+
+cloudDriver:
+  hubUrl: https://hub-cloud.browserstack.com/wd/hub
+  user: BROWSERSTACK_USERNAME
+  accessKey: BROWSERSTACK_ACCESSKEY
+  localTunnel:
+    enabled: true
+    local_options:
+      forcelocal: true
+      force: true
+      verbose: true
+      logFile: local_test.log
+  common_capabilities:
+    project: BrowserStack Demo Repository
+    buildPrefix: browserstack-examples-testng
```

Comment:
change to **browserstack-examples-cucumber-testng**

Reply:
Corrected the Build Prefix name to browserstack-examples-cucumber-testng